Automattic/wp-calypso
# Update copy and connect flow for Blaze plugin on non-woo sites

Related to #blaze-ads/pull/19

## Proposed Changes

* As part of the work to extend the Blaze for WooCommerce plugin to non-WooCommerce sites. This PR updates related copies and properties to the Blaze Ads plugin

## Why are these changes being made?


* Update copy for Blaze plugin
* Rename Woo DNA key from `blaze-ads` to `blaze-ads-on-woo` for Blaze Ads running on Woo
* Enable Jetpack magic link flow for Blaze on non-Woo sites

## Testing Instructions



* Checkout to D158244-code patch on your sandbox
* Point `widgets.wp.com` to your WP Sandbox.
* Run `yarn start` to start calypso 
* cd `apps/blaze-dashboard` and RUN `yarn dev --sync` to sync changes to your dashboard.
* In the blaze-ads plugin repo, set `define( 'WOOCOMMERCE_CALYPSO_ENVIRONMENT', 'development' );` in `blaze-ads.php`
* Generate the release plugin build from the blaze-ads plugin repo and install it in your test blog OR run locally as described in the repo readme.

Test for Woo sites
* Create a WooCommerce site, and make sure the Jetpack plugin is deactivated or not installed and Jetpack is disconnected
* In WP Admin, Go to `Marketing` -> `Blaze Ads`
* You should see the updated "Blaze Ads" name on the connect page
* Click the `Connect now` button to see the updated name in the Woo DNA-based Jetpack connect flow.
* Clicking the  "Get help setting up Blaze Ads" link should take you to the Woo documentation site

<img width="1417" alt="plugin dashboard" src="https://github.com/user-attachments/assets/00737df3-9fbc-4232-b26c-b24ecd6a497d">

<img width="1622" alt="jp connect" src="https://github.com/user-attachments/assets/59ab40f7-355d-41a8-aa9f-55370195c084">

Test for non-Woo sites
* In your test site, make sure WooCommerce is not installed/set up, the Jetpack plugin is deactivated or not installed and Jetpack is disconnected
* On your test site, ensure that WooCommerce is not set up and that the Jetpack plugin is either deactivated or not installed. Additionally, confirm that Jetpack is disconnected from your site.
* In WP Admin, Go to `Tools` -> `Advertising`
* You should see the updated "Blaze Ads" name on the connect page
* Click the `Connect now` button should take you to the default Jetpack connect flow.

<img width="1416" alt="jetpack logged in" src="https://github.com/user-attachments/assets/11fef7fc-af99-493e-80a7-67bb660b6bd7">




For non-loggedin/new users, they should see the magic link flow.

<img width="1728" alt="jetpack magic link" src="https://github.com/user-attachments/assets/0766d3a3-868c-41df-b87d-08fcde64b663">
<img width="941" alt="jetpack magic link confirmation" src="https://github.com/user-attachments/assets/1bf47b3e-495b-4356-b3f8-ce204339653b">
<img width="1388" alt="jetpack magic link email" src="https://github.com/user-attachments/assets/3c1a301c-4347-4c45-9f5e-d7d3dae1102c">



## Pre-merge Checklist
